rust/hyperlane-core/Cargo.toml

```
@@ -33,6 +33,7 @@ thiserror.workspace = true
 # version determined by ethers-rs
 primitive-types = "*"
 lazy_static = "*"
+derive_more = "0.99.17"
```

```suggestion
derive_more.workspace = true
```

Also please sort alphabetically